server.js

```diff
@@ -35,27 +35,32 @@ const locales = chokidar.watch(path.join(__dirname, 'locale'), opts)
 const layouts = chokidar.watch(path.join(__dirname, 'layouts'), opts)
 const statics = chokidar.watch(path.join(__dirname, 'static'), opts)
 
+const OTHER_SUPPORTED_LANGS = new Set(['ar', 'ca', 'de', 'es', 'fr', 'gl', 'it', 'ja', 'ko', 'uk', 'zh-cn', 'zh-tw'])
+
```

Comment:
It'd be better to use `fs` to get the directories under `locale` instead of fixed values.

Reply:
Good idea, I forgot and I'll fix that :)

Reply:
Fixed, many thanks!